continuousintegration.py

```diff
@@ -38,11 +38,17 @@ def watch_ci():
                   'Content-Length: 2\n' +
                   '\nOK\n')
         conn.close()
-        r = requests.get('https://api.github.com/repos/Charcoal-SE/SmokeDetector/git/refs/heads/master')
-        latest_sha = r.json()["object"]["sha"]
+        request = requests.get(
+            'https://api.github.com/repos/Charcoal-SE/SmokeDetector/git/refs/heads/master'
```

Comment:
This seems like a valid point where the line can go over 80 chars - it's one argument, so newlining it is weird, and it's a URL, so it doesn't make sense to split it.